test/emulator_backend.go

```diff
@@ -474,6 +474,18 @@ func (e *EmulatorBackend) DeployContract(
 }
 
 func (e *EmulatorBackend) ReadFile(path string) (string, error) {
+	// These are the scripts/transactions used by the
+	// BlockchainHelpers file.
+	if path == "mint_flow.cdc" {
```

Comment:
This might be a bit surprising for users if they have their own scripts with these file names. In that case, the built-in script would be used silently instead of their custom script, which might be hard to debug if the users script differs.

Maybe there is a better way to provide the helper scripts than using `Test.readFile`, but as you tried before, embedding them as string literals inside of the Cadence code is cumbersome / inelegant.

Maybe at least reduce the likelihood of / preventing a clash by making the path impossible to represent on the file system, by using a "path" that is illegal. For example, the `\0` (NULL) character could be an option. Maybe prefix with something like `"\u{0}helper/"` (in Cadence; in Go: `"\0helper/"`)

Reply:
Fair point, I was thinking of adding something like a unix timestamp :sweat_smile:

Reply:
Fixed in https://github.com/onflow/cadence-tools/pull/181/commits/2087d1397e96f0db9a31238d4e01fabd6143d190